training/creating_dataset.py

```diff
@@ -92,13 +283,64 @@ def load_and_preprocess_data(
     df = bestiary[raw_names]
     df.columns = target_names
 
+    for resistance in resistances:
+        with pd.option_context("mode.chained_assignment", None):
+            # silent warning (SettingWithCopyWarning) about view and copy
+            # we don't need to go back to the original df - no matter if it is a view
+            df[resistance] = bestiary[RESISTANCE_PATH].apply(
+                lambda x: get_characteristic_from_list(
+                    cell_value=x,
+                    characteristic_name=resistance.replace("_resistance", ""),
+                )
+            )
```

Comment:
Do tego i poniższych:
1. Pominąłbym podawanie explicite `cell_value` i `characteristic_name` - myślę, że to jest jasne z kontekstu
2. `characteristic_name=resistance.replace("_resistance", "")` dałbym do osobnej linijki
3. Lambdę też dałbym do osobnej linijki

Wtedy zostaje:
```
characteristic_name = resistance.replace("_resistance", "")
get_value = lambda x: get_characteristic_from_list(x, characteristic_name)
df[resistance] = bestiary[RESISTANCE_PATH].apply(get_value)
```